new-admin/src/views/layerforms/vectorlayerform.jsx

```diff
@@ -144,6 +173,8 @@ class VectorLayerForm extends React.Component {
     if (fieldName === "date") value = create_date();
```

Comment:
Please use camelCase for consistency. createDate

Reply:
Hi @jesade-vbg, 
Thanks for reviewing!
"create_date" is used in other *layerform.jsx and in search and edit.jsx so i suggest we leave it as is, for consinstency ;-).
I changed "rgba_to_string" to "rgbaToString" as you suggested in previous comment. Pushed it to github.

Reply:
Great! Sorry I was'nt paying attention to where create_date was placed. Admin is a diffrent story.